cpp/benchmarks/bench_simulation.cpp

```diff
@@ -0,0 +1,56 @@
+/* 
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: Rene Schmieding
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "bench_core.h"
+#include "bench_model.h"
+
+// google's benchmark templates require a function in the global scope, hence the following "using" statement
+using mio::benchmark::simulation;

```

Comment:
why don't you just put the whole function here? they are not reused anyway. And I would argue that the benchmark function should be close to the `BENCHMARK_...` declarations.

Same for mio::benchmark::integrator_step of course.

Reply:
Moved benchmarks functions above their corresponding benchmarks